samples/bluetooth/central_hr_coded/src/main.c

```diff
@@ -0,0 +1,279 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
```

Comment:
I am not sure that the licence is valid here. Could you confirm @joerchan or @carlescufi ?

Reply:
The license will have to be changed

Reply:
To what do I need to change the license?